### ISSUE TITLE
Literature Review and Summary of Model Transformation Languages

### ISSUE THREAD

**Issue by smiths** [JacquesCarette/Drasil#503]
The Drasil project would be improved by a greater understanding of Model Transformation Languages. Can you please start filling in information at the following wiki:

https://github.com/JacquesCarette/Drasil/wiki/Model-Transformation-Languages

You can start with a literature search. Any examples are good to start with. Don't worry about maintaining a clear narrative. To start with just collect all resources that are worthy of investigation.

**Comment by niazim3** [comment]
@smiths Was this change correct? https://github.com/JacquesCarette/Drasil/wiki/Model-Transformation-Languages/_compare/0481370146521f5e83e3330adf8f305f45b5afd0 (i.e. MDL -> MTL)

(Asking as this may be a convention I'm unaware of)

**Comment by niazim3** [comment]
If I find a pdf that I'm summarizing/extracting main points out of for now, should I be adding it to the repo right away or is linking the contents to the source enough for now? I'm asking because once this "research phase" is complete, there may be items that are not as significant/relevant as other information that's found...

**Comment by smiths** [comment]
You can hold off on adding a pdf to the repo until you are sure that it will be useful.  If you prematurely add a pdf and later decide not to use it, that isn't a big deal; it just take a bit of extra storage space that really doesn't effect us.   However, adding pdf files that we don't need means you are spending extra time on something that isn't that important.  If you copy every pdf that has the potential to be interesting into the repo, you will be spending too much time on that task.  (It also isn't a very fun task.)  😄

**Comment by JacquesCarette** [comment]
This feels like something that maybe @oluowoj could update?

**Comment by smiths** [comment]
Yes, I agree @JacquesCarette.  The first question for @oluowoj is the difference between MTLs and MDE.  My guess is that MTLs are a tool used for MDE, but I don't actually know.

**Comment by oluowoj** [comment]
Yes I will update this and provide more information on MTL. It is one of the concepts of MDE used in transforming models to other artifacts like documentation, source code , etc. I read the document by niazim, don't quite agree with some of the texts, I have added this to my to do list and will get back to it.

**Comment by JacquesCarette** [comment]
As MDE really is not so central to your work anymore, I'm unassigning @oluowoj from this.
